Aventra-MyEID-PKI-card.md

```diff
@@ -0,0 +1,150 @@
+The card is a JavaCard with Aventra MyEID applet that implements the functionality.
+
+The card material is PVC as standard, making it suitable for visual personalization using thermal transfer or dye sublimation printers.
+Customer specific layouts can be delivered in offset and silk screen printing.
+Optional features include magnetic stripe, signature panel, holograms, security printing etc.
+
+The cards can be personalized both visually and electrically by Aventra according to customer specifications, or the customers can personalize the cards themselves using ActivePerso Manager developed by Aventra, or software from other parties.
+
+MyEID is certified by Microsoft and supports Smart Card Plug and Play.
+
+>  MyEID version 4 has been released, adding support for Elliptic Curve Cryptography and many other new features.
+
+## Aventra MyEID PKI applet
+
+The MyEID applet implements all the basic functionality of a Public Key Infrastructure (PKI) token specified in the most common international PKI standards, such as PKCS#15. Users optionally have a choice between different authentication methods to the token. Besides the standard PIN number, there are currently two other authentication mechanisms available. The GrIDsure® one time PIN is based on a pop-up challenge grid that is used to form a onetime PIN that cannot be used by outsiders watching the authentication. MyEID tokens are also compatible with PalmSecure™ biometric technology, which is based on the unique blood vein patterns in the palm of the user’s hand replacing the PIN.
+
+### Technical details
+
+#### Platform
+
+* JavaCard™ from 2.2.1 and above, Global Platform 2.1.1
+
+#### Supported standards and specifications
+
+* ISO/IEC 7816-4 to 7816-9
+* ISO/IEC 14443 T=CL and Mifare™
+* PKCS#15
+* FINEID S4-1 and S4-2
+
+#### Other features
+
+* 512 bit to 4096 bit RSA cryptographic operations with on card key generation,
+* Secure random number generator (FIPS 140-2),
+* DES, 3DES, AES128, AES256 symmetric encryption algorithms,
+* 144K EEPROM memory,
+* Since MyEID 4: ECDSA and ECDH operations.
+
+#### Compatible software
+
+* OpenSC
+* Aventra MyEID Minidriver for Windows
+* Fujitsu mPollux DigiSign™ middleware
+* Large number of third party software products that support Microsoft Cryptography API: Next Generation (CNG) or PKCS#11 Token Interface
+
+## OpenSC support
+
+OpenSC 0.11.4 was the first version that had support for the MyEID card. At that time the patch required was provided by Aventra when requested. Since the version 0.11.10 support for the MyEID card is included to the official release. OpenSC initialization is supported from version 0.12.
+
+MyEID supports 1024 and 2048 bit RSA keys and EC keys in OpenSC. In OpenSC only normal PIN codes can be used. GrIDsure® and PalmSecure technologies are not supported. These require Aventra ActiveSecurity MyClient software.
```

Comment:
This is inconsistent with the rest of the document and with the code -- 3k and 4k is supported now too.